.github/CODEOWNERS

```diff
@@ -243,6 +244,7 @@
 /packages/kbn-optimizer/ @elastic/kibana-operations
 /packages/kbn-plugin-discovery/ @elastic/kibana-operations
 /packages/kbn-pm/ @elastic/kibana-operations
+/packages/kbn-shared-svg/ @elastic/kibana-operations
```

Comment:
Should the operations team be a CODEOWNER here? why?

Reply:
@spalger @mistic is there a reason why the ops team would need to look at changes in this package? It seems to me like any changes to this package are not of particular interest to you?

Reply:
I figured it was intentional and "allowed" it, but it feels like it was more of a test, which I failed 😅

I have no idea what this package is or why we would/wouldn't need to help with reviews. All I know is that based on the name it sounds like this package is going to contain "all of the shared svg in the entire repository" which sounds like a really bad idea, but I'm naming can easily be changed down the road.

Reply:
Hi @spalger, I can put the operations team out of the review, and will think about the name, not sure if it can be easily changed.